test/e2e/specs/editor/blocks/paragraph.spec.js

```diff
@@ -61,14 +63,7 @@ test.describe( 'Paragraph', () => {
 			editor,
 			pageUtils,
 			draggingUtils,
-			page,
 		} ) => {
-			await page.evaluate( () => {
-				window.wp.blocks.registerBlockType( 'test/v2', {
-					apiVersion: '2',
-					title: 'test',
-				} );
-			} );
```

Comment:
I assume there's no longer a need for a legacy canvas switch. There're some left overs, so removal is appreciated.

Reply:
Some tests still fail when running against iframed canvas, so I'd leave the switch. I tried migrating most of the legacy tests to iframed canvas, but it wasn't possible for some of them. For example, `cancels dragging blocks from the global inserter by pressing Escape` in `inserting-blocks.spec.js` does not hide the insertion point after pressing Escape, no matter what I do. It seems to be an issue with Playwright, because the tested functionality works as expected 100% of the time when testing manually.